File: samples_generated_code/samples_1/qwen/baseline_code_c/code_row_109.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int getMonthlySales() {
    return rand() % 100001;
}

int main() {
    srand(time(NULL));
    
    int monthlySales[12];
    for (int i = 0; i < 12; i++) {
        monthlySales[i] = getMonthlySales();
    }
    
    int q1Sales = monthlySales[0] + monthlySales[1] + monthlySales[2];
    int q2Sales = monthlySales[3] + monthlySales[4] + monthlySales[5];
    int q3Sales = monthlySales[6] + monthlySales[7] + monthlySales[8];
    int q4Sales = monthlySales[9] + monthlySales[10] + monthlySales[11];
    
    printf("Q1 Sales: %d\n", q1Sales);
    printf("Q2 Sales: %d\n", q2Sales);
    printf("Q3 Sales: %d\n", q3Sales);
    printf("Q4 Sales: %d\n", q4Sales);
    
    return 0;
}

File: samples_generated_code/sample_1/qwen/baseline_code_java/code_row_109.java
import java.util.Random;

public class SalesCalculator {

    public static void main(String[] args) {
        int q1Sales = 0;
        int q2Sales = 0;
        int q3Sales = 0;
        int q4Sales = 0;

        for (int month = 1; month <= 12; month++) {
            int sales = getMonthlySales();
            if (month >= 1 && month <= 3) {
                q1Sales += sales;
            } else if (month >= 4 && month <= 6) {
                q2Sales += sales;
            } else if (month >= 7 && month <= 9) {
                q3Sales += sales;
            } else {
                q4Sales += sales;
            }
        }

        System.out.println("Q1 Sales: " + q1Sales);
        System.out.println("Q2 Sales: " + q2Sales);
        System.out.println("Q3 Sales: " + q3Sales);
        System.out.println("Q4 Sales: " + q4Sales);
    }

    public static int getMonthlySales() {
        Random random = new Random();
        return random.nextInt(100001);
    }
}